docsource/modules150-160.rst

```diff
@@ -534,7 +534,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| pad                                       |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| pad_project                               |                      |                                                 |
+| |del| pad_project                               |Nothing to do         |Obsolete module, use directly in odoo itself     |
```

Comment:
```suggestion
| |del| pad_project                               | Done         |Obsolete module, use directly in odoo itself     |
```


Nothing to do should be used when when there is nothing to do ;-)

Reply:
ok i see, done, and also fix the related `pad` module too